# 实时功率热量数据

Header header

# 电压,单位伏
float64 voltage
# 电流,单位安
float64 current
# 功率,单位瓦
float64 power
# 功率缓冲,单位焦耳
int32 buffer
# 枪口热量数据
rm_referee_controller/PowerHeatShooter shooterHeat